src/RileyMaze/userFunction.js

```diff
@@ -0,0 +1,244 @@
+/* exported RMZUserFunction */
+
+const {GObject, Gtk} = imports.gi;
+
+const {Codeview} = imports.codeview;
+
+const FORWARD = 1;
+const UP = 2;
+const DOWN = 3;
+const JUMP = 4;
+const RILEY = {
+    // Keep this in sync with riley class in
+    // hack-toy-apps/com.endlessm.RileyMaze/parameters.js
+    queue: [],
```

Comment:
There was supposed to be an accompanying `ip` variable according to the API document, but I went ahead and just deleted that from the API document. It was secret anyway, and it's not actually needed.